bubble_sort.rb

```diff
@@ -0,0 +1,42 @@
+def bubble_sort(array)
```

Comment:
Bubble_sort should sort an array of numbers in increasing order, consider the following example:
```
irb> arr = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
irb> p arr.sort
irb> [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
irb> p bubble_sort(arr)
irb> 10
```
If I run the following code, it should return exactly the same as If I sort the array with the sort method.
![Screen Shot 2020-07-28 at 10 50 21 PM](https://user-images.githubusercontent.com/52430951/88754676-e50bb900-d124-11ea-89e6-93d9c5f6bfe4.png)

![Screen Shot 2020-07-28 at 10 50 35 PM](https://user-images.githubusercontent.com/52430951/88754686-eb019a00-d124-11ea-86af-d96a787197f7.png)

As you can see, the method should return an entire array, for now, it only returns the last value inside the array.